accounts-db/src/accounts_db.rs

```diff
@@ -11045,26 +11121,53 @@ pub mod tests {
         sample_storage_with_entries_id(tf, write_version, slot, pubkey, 0, mark_alive, None)
     }
 
-    fn sample_storage_with_entries_id(
+    fn sample_storage_with_entries_id_fill_percentage(
         tf: &TempFile,
         write_version: StoredMetaWriteVersion,
         slot: Slot,
         pubkey: &Pubkey,
         id: AppendVecId,
         mark_alive: bool,
         account_data_size: Option<u64>,
+        fill_percentage: u64,
     ) -> Arc<AccountStorageEntry> {
         let (_temp_dirs, paths) = get_temp_accounts_paths(1).unwrap();
-        let size: usize = aligned_stored_size(account_data_size.unwrap_or(123) as usize);
-        let mut data = AccountStorageEntry::new(&paths[0], slot, id, size as u64);
-        let av = AccountsFile::AppendVec(AppendVec::new(&tf.path, true, (1024 * 1024).max(size)));
+        let file_size = account_data_size.unwrap_or(123) * 100 / fill_percentage;
```

Comment:
@HaoranYi why is this `* 100 / fill_percentage`. I'm struggling to get the math here.

Reply:
$\dfrac{account \textunderscore data \textunderscore size} {file \textunderscore size} = \frac{ fill \textunderscore percentage} {100}$.

$file \textunderscore size = account \textunderscore data \textunderscore size \cdot \dfrac{100}{fill \textunderscore percentage}$